edweiss-app/__test__/utils/utilsFunctions.test.ts

```diff
@@ -85,7 +86,6 @@ describe('statusNextAction', () => {
 
 describe('toogleArchivityOfTodo', () => {
     test('should archive todo and show success toast on success', async () => {
-        const archivedTodo = { ...mockTodo, status: 'archived' };
         (callFunction as jest.Mock).mockResolvedValueOnce({ status: true });
```

Comment:
Hey, that could be solution. However, my approach was to create only one mock (applying the economy of mechanism principle 😊). This approach allow to only change the status field to archive so that it's easier to understand that they are the same only the status differs.